packages/mintlist-cli/src/cmd/bump.ts

```diff
@@ -0,0 +1,183 @@
+import { Mintlist } from "@orca-so/token-sdk";
+import { execSync } from "child_process";
+import { MetadataOverrides, MintlistFileUtil } from "../util/mintlist-file-util";
+
+interface BumpOptions {
+  before: string;
+  after: string;
+}
+
+type VersionChange = "major" | "minor" | "patch";
+
+/**
+ * Bumps the version of the package based on the changes in the mintlists.
+ * If a mintlist is added or removed, a major version bump is performed.
+ * If the only change is added mints, a minor version bump is performed.
+ * If the only change is removed mints or updated overrides, a patch version bump is performed.
+ */
+export function bump({ before, after }: BumpOptions) {
+  if (!hasDeps()) {
+    console.log("npm and git must be installed and in the PATH");
+    process.exit(1);
+  }
+
+  if (hasUncommittedChanges()) {
+    console.log("Must have clear working directory");
+    process.exit(1);
+  }
+
+  const versionChange = getVersionChange(before, after);
+  if (!versionChange) {
+    return;
+  }
+
+  const version = execSync(`npm version ${versionChange}`, { encoding: "utf-8" }).trim();
+  console.log(`${version}`);
+}
+
+function getVersionChange(before: string, after: string): VersionChange | undefined {
+  if (hasMintlistChanges(before, after)) {
+    return "major";
+  }
+  const { hasAdded, hasRemoved } = diffMintlists(before, after);
+  if (hasRemoved) {
+    return "minor";
+  }
+  if (hasAdded || diffOverrides(before, after)) {
+    return "patch";
+  }
+}
+
+function diffOverrides(before: string, after: string): boolean {
+  const beforeOverrides = MintlistFileUtil.fromString<MetadataOverrides>(
+    execSync(`git show ${before}:src/overrides.json`, {
```

Comment:
I think the flexibility of the options of gen-tokenlist makes the tool a bit complicated. How about making it a rule that src/overrides.json must be created for this tool, and removing the overrides option from the gen-tokenlist command.